functions/analytics.js

```diff
@@ -122,11 +166,11 @@ exports.generateAnalytics = functions.https.onCall(async (data, context) => {
       const { user, visits } = userWithVisits;
       const { id, partyName } = user;
       const { email } = authUsersById[id] || {};
-      const visitIds = visits.map((el) => el.id);
+      // const visitIds = visits.map((el) => el.id);
 
-      visitIds.map((visit) =>
-        fs.writeFileSync(allSpaceVisitsFileName, `${visit} \n`, { flag: "a" })
-      );
+      // visitIds.map((visit) =>
+      // fs.writeFileSync(allSpaceVisitsFileName, `${visit} \n`, { flag: "a" })
+      // );
```

Comment:
These kinds of comments tend to drift away with time. Might be easier to just delete the lines.

https://github.com/sparkletown/sparkle/blob/redesign-2021/docs/code-style.md#maybe-could-be-useful-later-dead-code